File: app/src/main/java/com/example/config/RemoteConfig.java
package com.example.config;

import com.example.BuildConfig;
import com.google.firebase.remoteconfig.FirebaseRemoteConfig;
import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings;

public final class RemoteConfig {
	public static void setup() {
		final FirebaseRemoteConfigSettings remoteConfigSettings =
			new FirebaseRemoteConfigSettings
				.Builder()
				.setDeveloperModeEnabled(BuildConfig.DEBUG)
				.build();

		FirebaseRemoteConfig
			.getInstance()
			.setConfigSettings(remoteConfigSettings);
	}
}

File: app/src/main/kotlin/br/com/vitorsalgado/example/config/RemoteConfig.kt
package br.com.vitorsalgado.example.config

import br.com.vitorsalgado.example.BuildConfig
import com.google.firebase.remoteconfig.FirebaseRemoteConfig
import com.google.firebase.remoteconfig.FirebaseRemoteConfigSettings

object RemoteConfig {
  fun setup() =
    FirebaseRemoteConfig
      .getInstance()
      .setConfigSettings(
        FirebaseRemoteConfigSettings.Builder()
          .setDeveloperModeEnabled(BuildConfig.DEBUG)
          .build()
      )
}
